src/backend/base/langflow/services/database/models/mcp/model.py

```diff
@@ -0,0 +1,21 @@
+from uuid import UUID
+
+from pydantic import BaseModel
+
+
+class MCPSettings(BaseModel):
```

Comment:
May I ask why this was added?

Reply:
Oh.. I think I understand it now. This should probably be moved to the schema in the `api` module.